app/rtc_s1_parallel.py

```diff
@@ -0,0 +1,887 @@
+
+            sub_swaths.set_valid_samples_array(1, valid_samples_sub_swath)
+
+            # geocode
+
+        else:
+            sub_swaths = None
+
+        if flag_geocoding_without_sub_swaths:
+            # geocode (without sub_swaths)
+            
+            if flag_inform_user_about_sub_swaths_error:
+                logger.warning('WARNING the sub-swath masking is not available'
+                               ' from this ISCE3 version. The sub-swath masking'
+                               ' was disabled.')
+
+    
+        # Output imagery list contains multi-band files that
+        # will be used for mosaicking
+        output_imagery_list.append(geo_burst_filename)
+
+        # If burst imagery is not temporary, separate polarization channels
+        if not flag_bursts_files_are_temporary:
+            output_burst_imagery_list = []
+            for pol in pol_list:
+                geo_burst_pol_filename = \
+                    os.path.join(output_dir, burst_id,
+                        f'{product_prefix}_{pol}.' +
+                        f'{imagery_extension}')
+                output_burst_imagery_list.append(geo_burst_pol_filename)
+
+            if os.path.exists(geo_burst_filename):
+                rtc_s1._separate_pol_channels(geo_burst_filename,
+                                       output_burst_imagery_list,
+                                       logger, output_raster_format)
+            else:
+                for filename in output_burst_imagery_list:
+                    os.rename(filename.replace(output_dir, scratch_path), filename)
+
+                # create a vrt from the separated pol tiffs
+                geo_burst_filename +='.vrt'
+                output_imagery_list[-1] = geo_burst_filename
+                gdal.BuildVRT(geo_burst_filename, output_burst_imagery_list,
+                              options=vrt_options_mosaic)
+
+            output_file_list += output_burst_imagery_list
+
+        if save_nlooks:
+            if not flag_bursts_secondary_files_are_temporary:
+                logger.info(f'file saved: {nlooks_file}')
+            output_metadata_dict['nlooks'][1].append(nlooks_file)
+    
+        if save_rtc_anf:
+            if not flag_bursts_secondary_files_are_temporary:
+                logger.info(f'file saved: {rtc_anf_file}')
+            output_metadata_dict['rtc'][1].append(rtc_anf_file)
+
+        radar_grid_file_dict = {}
+        if flag_call_radar_grid and save_bursts:
+            rtc_s1.get_radar_grid(
+                geogrid, dem_interp_method_enum, product_prefix,
+                bursts_output_dir, imagery_extension, save_incidence_angle,
+                save_local_inc_angle, save_projection_angle,
+                save_rtc_anf_psi,
+                save_range_slope, save_dem,
+                dem_raster, radar_grid_file_dict,
+                mosaic_geogrid_dict, orbit,
+                verbose = not flag_bursts_secondary_files_are_temporary)
```

Comment:
I think we need some sort of way to populate the dict `radar_grid_file_dict` which is right before the code snippet above. Otherwise, the incidence, angle, etc. will not be saved after the parent process even we turn the flag on in runconfig?

Currently the dict is populated by `get_radar_grid()`, more precisely  `get_radar_grid() -> _create_raster_obj()`

I can copy and paste the naming convention what is inside `get_radar_grid()` to populate `radar_grid_file_dict`. Any other good idea?